README.md

```diff
@@ -44,8 +44,28 @@ There are more features which can be explored using various examples for the dif
   - `conda activate <name-of-choice>`
 - Install all necessary packages using `pip install -r requirements.dev.txt` in the project root
 
+### SDKs
+
+- Download or Build
+  - Run `yarn sdk.get` to download the most recent prebuilt SDK releases from GitHub __without__ needing to build them from scratch. You can continue with the installation. You will find the downloaded artifacts in their respective folder _./src/sdk/(javascript,python,vscode)/lib_
+  - JS SDK _(mandatory)_
```

Comment:
I think it is only mandatory for js-talents or to run platform components locally, or?
For example: If you only want to implement a c/c++-talent and start the platform via docker-compose that it is not needed, or?

Maybe we should mention the use case here (also for the other points below) --> js-talent development / running platform locally, c/c++-talent development...

Reply:
To make the VSCode Extension working you need to have at least one artifact on your disk. So I would propose it as mandatory. Especially if you are looking to do the "asap" steps.